lms/static/js/student_account/views/RegisterView.js

```diff
@@ -80,18 +80,25 @@
 
             saveSuccess: function() {
                 // Edraak (google-analytics): Send event on registration success
+                const k12_cookie = $.cookie('k12-landing')
+                var eventLabel = 'Universal'
+
+                if (k12_cookie) {
+                  eventLabel = 'K12'
+                }
+
```

Comment:
I think if you define this in the prerender clause it might be closer to DRY
`this.eventLabel = ($.cookie('k12-landing') ? 'K12' : 'Universal'`
@Salomari1987 let  me know what you think

Reply:
it's 100% fine the way it is as well, I'll leave it up to you to decide